README.md

```diff
@@ -59,33 +60,89 @@ Table of Contents
 
 Created by [gh-md-toc](https://github.com/ekalinin/github-markdown-toc)
 
-## Setup:
-The following `$SHELL` variables should be defined:
+## Setup
 
-- `$BUILD_HOME`
-- `$DATA_PATH`
-- `$ELOG_PATH`
+The `$SHELL` variable `$ELOG_PATH` should be defined:
+
+```
+export ELOG_PATH=/your/favorite/elog/path
+```
+
+Remove and download the setup script:
+
+```
+rm -f setup_gemdaq.sh
+wget https://raw.githubusercontent.com/cms-gem-daq-project/sw_utils/master/scripts/setup_gemdaq.sh
+```
 
 Then execute:
 
 ```
-source $BUILD_HOME/cmsgemos/setup/paths.sh
-source $BUILD_HOME/gem-plotting-tools/setup/paths.sh
+source setup.sh -c 0.3.1 -g 1.0.0 -G 5 -v 2.0.0 -V 3
 ```
 
-If this is the first time you are executing the above commands on `lxplus` it will create a `python v2.7` `virtualenv` for you.  It may take some time to download the necessary packages so be patient and do not interrupt the installation.  To disable this python env execute:
+If this is the first time you are executing the above command, it will create a Python `virtualenv` for you. It will also install the following packages:
+
+* `cmsgemos` version 0.3.1 (`-c 0.3.1`)
+* `gemplotting` version 1.0.0-dev5 (`-g 1.0.0 -G 5`)
+* `vfatqc` version 2.0.0-dev3 (`-v 2.0.0 -V 3`)
```

Comment:
Change too:

---
* [cmsgemos](https://github.com/cms-gem-daq-project/cmsgemos/tags) version X.Y.Z (`-c X.Y.Z`)
* [gemplotting](https://github.com/cms-gem-daq-project/gem-plotting-tools/tags) version X.Y.Z-dev**A** (`-g X.Y.Z -G A`)
* [vfatqc](https://github.com/cms-gem-daq-project/vfatqc-python-scripts/tags) version X.Y.Z-dev**A** (`-v X.Y.Z -V A`)

Note that if a dev version is not to be used (normal) drop the -G option.

---

Again hard coding a given release should be avoided (in anticipation of future changes).

Reply:
How do I connect it to the line above:

```
source setup_gemdaq.sh -c 0.3.1 -g 1.0.0 -G 5 -v 2.0.0 -V 3
```

Should I use `A.B.C`, `D.E.F-devG`, `H.I.J-devK` ? Or worse, `X1.X2.X3`, `Y1.Y2.Y3-devY4`, `Z1.Z2.Z3-devZ4` ? I seems to me that both are less readable than a concrete example.

Reply:
I am not asking you to remove the concrete example but to provide additional information in the form of a bullet point list with links to the relevant tag areas on GitHub with syntax shown above.

Reply:
If you meant "add" and not "change", where should I add it? How do I connect it to:

```
source setup.sh -c <cmsgemos tag> -g <gem-plotting tag> -G <gem-plotting dev version optional>
```

Sorry but I really don't understand how you want me to fit this new bullet list into the text.